test.js

```diff
@@ -1,12 +1,12 @@
-import { spy, stub } from 'sinon'
-import bodyParser from 'body-parser'
-import express from 'express'
-import delay from 'delay'
-import auth from 'basic-auth'
-import pify from 'pify'
-import test from 'ava'
-import Analytics from '.'
-import { version } from './package'
+const { spy, stub } = require('sinon')
```

Comment:
Annoyingly bable was removed from ava and imports arent native until node 13 so have to revert to requires here